_episodes/45.md

```diff
@@ -0,0 +1,258 @@
+
+## What is workflow orchestration?
+
+Workflow orchestration refers to the process of coordinating and automating
+complex workflows consisting of multiple interconnected tasks. This involves
+designing and defining the workflow, scheduling and executing the tasks,
+monitoring the progress and outcomes, and handling any errors or exceptions that
+may arise. In the context of Trino, the tasks can be generally thought of as a
+Trino query.
+
+<img src="/assets/episode/45/data-pipelines.png">
+
+## Why do we need a workflow orchestration tool for building a data lake?
+
+Building a data lake can involve many complex and interdependent data processing
+tasks, which can be challenging to manage and scale without a workflow
+orchestration tool. Sometimes we can consider tools like Trino at the center of 
+the universe, and perhaps it would be easier to schedule SQL queries with a much
+simpler tool. Most companies, however, require a larger variety of tasks to
+build a data lake that interoperate on more than just running SQL on Trino. Even
+if you primarily run Trino SQL scripts to run these jobs, it is better to have
+an orchestration tool instead of managing by hand.
+
+## What is Apache DolphinScheduler?
+
+<img width="75%" src="/assets/episode/45/dolphin-scheduler.png">
+
+Apache DolphinScheduler is an open-source, distributed workflow scheduling
+platform designed to manage and execute batch jobs, data pipelines, and ETL
+processes. DolphinScheduler enables users to create and manage successive jobs
+run easily, including support for different types of tasks, such as SQL
+statements, shell scripts, Spark, Kubernetes, and many others. In short, it's a
+powerful and user-friendly workflow orchestration platform that enables users to
+automate and manage their complex data processing tasks.
+
+Read [this blog on Trino and Apache DolphinScheduler](https://blog.devgenius.io/dolphinscheduler-helps-trino-quickly-realize-the-integrated-data-construction-of-lake-and-warehouse-cde095b6573b)
+to find out more.
+
+### Does DolphinScheduler have any computing engine or storage layer?
+
+DolphinScheduler is a powerful tool for managing and orchestrating data
+processing workflows across a range of computing engines and storage systems,
+but it does not provide its own computing or storage capabilities.
+
+## What are the differences with other workflow orchestration, such as Apache Airflow?
+
+Both Dolphin Scheduler and Airflow are designed to be scalable and highly
+available which supports large-scale distributed environments well.
+
+Airflow supports a wide range of third-party integrations, including popular
+data processing frameworks such as Trino, Spark, and Flink, as well as with
+cloud services such as AWS and GCP. Dolphin Scheduler supports a similar range
+of data processing frameworks and tools making both versatile platforms for
+managing diverse data processing tasks. Airflow is certainly ahead which makes
```

Comment:
Dont say that Airflow is ahead .. when we want to talk positively about DolphinScheduler